src/compile/data/formatparse.ts

```diff
@@ -113,7 +123,13 @@ export class ParseNode extends DataFlowNode {
     other.remove();
   }
   public assembleFormatParse() {
-    return this._parse;
+    const formatParse = {};
```

Comment:
Clarifying question:  when do we still use format parse? 

(It seems like for the most case, we use expression. -- maybe add comment to clarify?)